installation.md

```diff
@@ -1,27 +1,31 @@
-## Installation
+# Installation
 
-KubeVirt is a virtualization add-on to Kubernetes and this guide assumes that a Kubernetes cluster is already installed.
+KubeVirt is a virtualization add-on to Kubernetes and this guide assumes that a
+Kubernetes cluster is already installed.
 
-### Requirements
+## Requirements
 
 A few requirements need to be met before you can begin:
 
 * [Kubernets](https://kubernetes.io) cluster \([OpenShift](https://github.com/openshift/origin), Tectonic\)
 * kubectl client utility
 * git
 
-#### Minimum Requirements
+### Minimum Requirements
 
 | Component | Minimum Version |
 | --- | --- |
 | Kubernetes | 1.7 |
 | KubeVirt | 0.0.3 |
```

Comment:
This should probably be 0.1.0-alpha due to `https://github.com/kubevirt/kubevirt/releases/download/${RELEASE}/kubevirt.yaml`